src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowVscTest.java

```diff
@@ -335,4 +337,53 @@ void testHvdcDirectionChangeAcEmulation() {
         assertTrue(pcs1 < 0, "Power delivered by cs1");
         assertTrue(Math.abs(pcs2) > Math.abs(pcs1), "Loss at HVDC output");
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"LCC", "VSC"})
+    void testHvdcDisconnectedInIDM(String testType) {
+        HvdcConverterStation.HvdcType hvdcType = switch (testType) {
+            case "LCC" -> HvdcConverterStation.HvdcType.LCC;
+            default -> HvdcConverterStation.HvdcType.VSC;
+        };
+
+        LoadFlowParameters parameters = new LoadFlowParameters();
+        parameters.setHvdcAcEmulation(
+            switch (testType) {
+                case "VSC-AcEmulation" -> true;
+                default -> false;
+            });
+
+        Network network = HvdcNetworkFactory.createHvdcLinkedByTwoLinesAndSwitch(hvdcType);
+
+        LoadFlow.Runner loadFlowRunner = new LoadFlow.Runner(new OpenLoadFlowProvider(new DenseMatrixFactory()));
+        LoadFlowResult result = loadFlowRunner.run(network, parameters);
+        assertTrue(result.isFullyConverged());
+
+        assertTrue(network.getGenerator("g1").getTerminal().getP() < -299.99, "Generator expected to deliver enough power for the load");
+        assertTrue(network.getGenerator("g1").getTerminal().getP() > -310, "Power loss should  be realistic");
```

Comment:
why not just lock to expected result?

`assertActivePowerEquals(-304.4, network.getGenerator("g1").getTerminal());`